lib/platform/gitlab/index.js

```diff
@@ -381,12 +381,93 @@ function addReviewers(iid, reviewers) {
   logger.warn('Unimplemented in GitLab: approvals');
 }
 
-async function ensureComment() {
-  // Todo: implement. See GitHub API for example
+async function getComments(issueNo) {
+  // GET https://gitlab.com/api/v4/projects/:owner/:repo/merge_requests/:number/notes
+  logger.debug(`Getting comments for #${issueNo}`);
+  const url = `https://gitlab.com/api/v4/projects/${config.parentRepo ||
```

Comment:
No need for `config.parentRepo ||` in GitLab - that's only needed for `forkMode`, which is GitHub-only. Please remove all instances you find.